runtime/interpreter/encoding_test.go

```diff
@@ -3097,7 +3097,7 @@ func TestEncodeDecodeCapabilityValue(t *testing.T) {
 			// positive integer to follow
 			0x18,
 			// public account (tag)
-			0x5b,
+			0x59,
```

Comment:
After fixing the primitive static type constants above, this should be reverted:

```suggestion
			0x5b,
```